src/test/lib/ApiTest.cpp

```diff
@@ -5980,549 +5980,322 @@ QuicTestCredentialLoad(const QUIC_CREDENTIAL_CONFIG* Config)
     TEST_QUIC_SUCCEEDED(Configuration.LoadCredential(Config));
 }
 
+
+class QuicStorageSettingScopeGuard {
+public:
+    static
+    QuicStorageSettingScopeGuard Create(
+        _In_opt_ const char* StorageName) {
```

Comment:
nit: since you use this with `nullptr`, it can be a nice default value to provide.

```suggestion
        _In_opt_ const char* StorageName = nullptr) {
```